### PR TITLE
Implement updated volume control overlay design

### LINKED ISSUE #1982: Volume Controls
volume-settings

This is how the volume control looks like for osu!lazer. Almost similar to stable's, but it's on the left side.  Subtle dark gradient from the left for visibility purposes when the controls are active.

Hover each circle to modify the values. Mute/Unmute button. Bright orange outline on hover. Changes to Mute icon when muted.

Ignore the button with the artistically red cross drawn on it.

### PR DESCRIPTION
**Pull request by jorolf**
Closes #1982
I'm not so sure if these are the correct colour / glow values

### CODE REVIEW

File: osu.Game/Overlays/Volume/MuteButton.cs [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+            CornerRadius = HEIGHT / 2;
+            Size = new Vector2(width, HEIGHT);
+        }
+
+        [BackgroundDependencyLoader]
+        private void load(OsuColour colours)
+        {
+            hoveredColour = colours.YellowDark;
+            BorderColour = unhoveredColour = colours.Gray1.Opacity(0.9f);
+
```
Comment (VPeruS) [inline osu.Game/Overlays/Volume/MuteButton.cs]: Empty line

File: osu.Game.Tests/Visual/TestCaseVolumePieces.cs [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        {
+            VolumeMeter meter;
+            Add(meter = new VolumeMeter("MASTER", 125, colours.PinkDarker));
+            Add(new MuteButton
+            {
+                Margin = new MarginPadding { Top = 200 }
+            });
+
+            meter.Bindable.BindTo(audio.Volume);
+
```
Comment (VPeruS) [inline osu.Game.Tests/Visual/TestCaseVolumePieces.cs]: Empty line

File: osu.Game/Overlays/VolumeOverlay.cs [hunk 3]
```diff
@@ -1,57 +1,91 @@
...
+                        muteButton = new MuteButton
+                        {
+                            Margin = new MarginPadding { Top = 100 }
+                        }
                     }
-                }
+                },
+            });
+
+
```
Comment (VPeruS) [inline osu.Game/Overlays/VolumeOverlay.cs]: Empty line

**Comment by peppy** [comment]
I made some minor changes, thinking I'd only have to make minor changes, but ended up tidying up the binding code. Make sure you check what I changed as it is a huge improvement over what you had.

Mostly good code though, nice job.

There's still missing functionality from this overlay, such as drag adjustment, potentially keeping it visible on hover and a bit more animation similar to stable so you are aware of the currently hovered/active control, but I think that's best added in future PRs.

**Review by peppy — APPROVED** [review]

**Pull request merged**